Makefile

```diff
@@ -9,7 +9,7 @@ LINTIGNOREPKGCOMMENT='service/[^/]+/doc_custom.go:.+package comment should be of
 UNIT_TEST_TAGS="example codegen awsinclude"
 
 SDK_WITH_VENDOR_PKGS=$(shell go list -tags ${UNIT_TEST_TAGS} ./... | grep -v "/vendor/src")
-SDK_ONLY_PKGS=$(shell go list ./... | grep -v "/vendor/")
+SDK_ONLY_PKGS=$(shell go list ./... | grep -v "/vendor/" | grep -v "/models/apis/)
```

Comment:
could also add a "stub.go" file to the models/apis folder, and i don't think this change would be needed.